### DISCUSSION TITLE
How to perform Word nagivation mapping

### DISCUSSION THREAD

**Discussion by rieger-jared** [kovidgoyal/kitty#3413] [General]
So I'm coming from Iterm, and mapping the sequence `^[b` which moves back one word and vice versa `^[f` doesn't seem to work with the configuration 

```txt
map alt+left send_text all \x1b b
```
I've tried several variations but I have no idea how to combine several keys together. Does anyone know how to do this? It's my finial block from fully adopting kitty.

**Comment by rieger-jared** [comment]
I would also add, something as simple as `map ctrl+delete send_text all \x15` where `\x15` is the equivilent of pressing `^u` doesn't work either

**Comment by trygveaa** [comment]
You need to remove the space between `\x1b` and `b` in your first mapping. Your second mapping should work fine, make sure to restart kitty after changing the config.

**Reply by rieger-jared** [reply]
Ah, great! Yeah, simply placing the char afterwards works. However, the `map ctrl+delete send_text all \x15` binding is still not working. kitty --debug-keyboard doesn't seem to yield much

**Reply by trygveaa** [reply]
`--debug-keyboard` should print info about the keys pressed, and also `Keypress matched action: send_text`. Try running `showkey -a` and pressing ctrl+delete. With that mapping it should print `^U 	21 0025 0x15`.
